keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -490,6 +505,13 @@ For GA, this section is required: approvers should be able to confirm the
 previous answers based on experience in the field.
 -->
 
+Without this feature, a user can forcefully delete the pods after they are
```

Comment:
Does the new logic add any additional expensive calls to the controllers? such as List Pods?

Reply:
There’s a Get Node call to check the node status and taint, but the new logic has not added new expensive calls, i.e., there isn't a new List Pods call.